torch/_inductor/decomposition.py

```diff
@@ -453,6 +453,17 @@ def dequantize_per_tensor_tensor_decomp_impl(
     return (input.to(torch.float32) - zero_point) * scale
 
 
+@register_decomposition(torch.ops.quantized.embedding_bag_byte_unpack)
+def q_embedding_bag_byte_unpack_decomp(packed):
+    def bitcast_u8_to_f32(u8):
+        x, y, z, w = [u8[..., n].to(torch.int32) for n in (0, 1, 2, 3)]
+        return (x + (y << 8) + (z << 16) + (w << 24)).view(torch.float32)[..., None]
+
+    scales = bitcast_u8_to_f32(packed[..., -8:-4])
```

Comment:
Directly view with `torch.float32` doesn't work? `packed[..., -8:-4].view(torch.float32)`.

Reply:
Unfortunately inductor doesn't support views with differing bitwidth yet, even though pytorch does.  You get a "bitcast to different bitwidth type is not supported yet" from lowering.py.  It would certainly be nice to support that at some point!